db/migrate/20230412013301_create_account_requests.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountRequests < ActiveRecord::Migration[7.0]
+  def change
+    create_table :account_requests do |t|
+      t.string :username
+      t.string :FullName
```

Comment:
It is recommended to utilize snake case when naming database attributes, such as `full_name`.